modules/Bio/EnsEMBL/IO/Parser.pm

```diff
@@ -297,9 +296,10 @@ sub _open_as {
       my $class = 'Bio::EnsEMBL::IO::Parser::'.$subclass;
       my $object;
 
-      try {
```

Comment:
Have you tried running this with the `require $class` rather than adding back all the `evals`?

Reply:
No, I just want it to work so I can get on with my actual tickets :)

Reply:
looking at it I'm pretty sure the Try::Tiny is not to blame, it has to be the removed `eval "require $class";`
Putting that back should get it working again, although it should not be necessary if the imports were done properly (doesn't look like it).